Added Fetch Key Vault Secret action (#67)
DFE-Digital/github-actions: fetch-key-vault-secrets/README.md

## Fetch Key Vault Secret
This action uses the az CLI to retrieve values of secrets that resides in the Azure Key Vault
[Usage](/.github/workflows/test-fetch-key-vault-secrets.yml)
### Usage
```yml
    - uses: azure/login@v1
      with:
        creds: ${{ secrets.AZURE_CREDENTIALS }}

    - name: Fetch Secrets

>         id: get-secret
>         uses: DFE-Digital/github-actions/fetch-key-vault-secrets@master # Replace with your action repo
>         with:
>           KEY_VAULT: your-keyvault-name # Replace with your key vault name
>           SECRETS: secret1,secret2,secret3 # Replace with your secret names
> 
>       - name: Use the secret value
>         run: echo "SECRET 1 = ${{ steps.get-secret.outputs.secret1 }}"
>         run: echo "SECRET 2 = ${{ steps.get-secret.outputs.secret2 }}"
>         run: echo "SECRET 3 = ${{ steps.get-secret.outputs.secret3 }}"

Reviewer: ```suggestion
      id: get-secret
      uses: DFE-Digital/github-actions/fetch-key-vault-secrets@master # Replace with your action repo
      with:
        KEY_VAULT: your-keyvault-name # Replace with your key vault name
        SECRETS: secret1,secret2,secret3 # Replace with your secret names

    - name: Use the secret value
      run: echo "SECRET 1 = ${{ steps.get-secret.outputs.secret1 }}"
      run: echo "SECRET 2 = ${{ steps.get-secret.outputs.secret2 }}"
      run: echo "SECRET 3 = ${{ steps.get-secret.outputs.secret3 }}"
```

# Fetch Key Vault Secrets

Fetch Secrets from the Azure Key vault taking Key Vault name and secret names as input.

## Inputs

> - `KEY_VAULT`: Name of the Key Vault containing the secrets (Required)
> - `SECRETS`: Comma seperated secret names that are going to be fetched (Required)

Reviewer: ```suggestion
- `keyvault`: Name of the Key Vault containing the secrets (Required)
- `secrets`: Comma separated secret names that are going to be fetched (Required)
```

> - `KEY_VAULT`: Name of the Key Vault containing the secrets (Required)
> - `SECRETS`: Comma separated secret names that are going to be fetched (Required)

Reviewer: ```suggestion
- `keyvault`: Name of the Key Vault containing the secrets (Required)
- `secrets`: Comma separated secret names that are going to be fetched (Required)
```

## Outputs

> - `SECRET_<secretname>`: Masked Value of the secret

Reviewer: ```suggestion
- `SECRET_<secretname>`: Value of the secret
```

The masked value would be just `x`s I think, whereas this should contain the real value.

## Example

```yml
  - uses: azure/login@v1
    with:
      creds: ${{ secrets.AZURE_CREDENTIALS }}

>   - name: Fetch Secrets
>       id: get-secret
>       uses: DFE-Digital/github-actions/fetch-key-vault-secrets@master # Replace with your action repo
>       with:
>         keyvault: your-keyvault-name # Replace with your key vault name
>         secrets: secret1,secret2,secret3 # Replace with your secret names
> 
>     - name: Use the secret value
>       run: echo "SECRET 1 = ${{ steps.get-secret.outputs.SECRET_secret1 }}"
>       run: echo "SECRET 2 = ${{ steps.get-secret.outputs.SECRET_secret2 }}"
>       run: echo "SECRET 3 = ${{ steps.get-secret.outputs.SECRET_secret3 }}"

Reviewer: ```suggestion
  - name: Fetch Secrets
    id: get-secret
    uses: DFE-Digital/github-actions/fetch-key-vault-secrets@master # Replace with your action repo
    with:
      keyvault: your-keyvault-name # Replace with your key vault name
      secrets: secret1,secret2,secret3 # Replace with your secret names

  - name: Use the secret value
    run: echo "SECRET 1 = ${{ steps.get-secret.outputs.SECRET_secret1 }}"
    run: echo "SECRET 2 = ${{ steps.get-secret.outputs.SECRET_secret2 }}"
    run: echo "SECRET 3 = ${{ steps.get-secret.outputs.SECRET_secret3 }}"
```

The indentation in the example is still slightly wrong, after line 20 everything is indented by 2 spaces more than it should be.
